static/sass/styles.scss

```diff
@@ -490,4 +490,37 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
 /// can be removed once added upstream
 .is-paper {
   background-color: #f3f3f3;
-}
\ No newline at end of file
+}
+
+.p-department-cards__container {
+  aspect-ratio: 2 / 3;
+  background-color: #fff;
+  margin-bottom: $spv--large;
+  margin-top: $spv--small;
+
+  .p-department-cards__card {
```

Comment:
In Vanilla we have `%vf-card` and similar `%vf-card-padding` (with smaller scope) that can be extended to get all the shared styles of card pattern. I guess this could be used here to avoid repeating paddings, background, etc.

<img width="770" alt="image" src="https://github.com/canonical/canonical.com/assets/83575/41fe56f0-4104-4dc8-aa93-dd99e10554a1">